app/views/vzory/prieskum-spokojnosti.njk

```diff
@@ -53,7 +53,8 @@
             prolog: 'Cieľom tohto dotazníka je zistiť potreby a postrehy používateľov webu Ministerstva investícií, regionálneho rozvoja a informatizácie (Ďalej len "MIRRI").'
           },
           {
-            prolog: "Ďakujeme za súhlas s účasťou v tomto prieskume, pomáhate nám zlepšovať tento web. Vyplnenie tohto dotazníka Vám zaberie len 3 minúty."
+            prolog: "Ďakujeme za súhlas s účasťou v tomto prieskume, pomáhate nám zlepšovať tento web. Vyplnenie tohto dotazníka Vám zaberie ",
+            boldItem: "len 3 minúty."
```

Comment:
toto netreba robit takto.. kludne to moze byt 
` prolog: "Ďakujeme za súhlas s účasťou v tomto prieskume, pomáhate nám zlepšovať tento web. Vyplnenie tohto dotazníka Vám zaberie <strong>len 3 minúty.</strong>"`

a v template to bude 
`      {{ params.prolog | safe }}`